documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -13,7 +13,11 @@
 
 ### Dependency updates
 
-* Upgraded Spring Boot to version 2.7.12 to resolve [CVE-2023-20883](https://nvd.nist.gov/vuln/detail/CVE-2023-20883)
+* Upgraded Spring Boot to version 2.7.12 to resolve [CVE-2023-20883](https://nvd.nist.gov/vuln/detail/CVE-2023-20883) 
+* Upgraded jackson-databind to version 2.15.0 to resolve high severity [CVE-2022-42003](https://nvd.nist.gov/vuln/detail/CVE-2022-42003) and high severity [CVE-2022-42004](https://nvd.nist.gov/vuln/detail/CVE-2022-42004)
```

Comment:
Hi Shanty, the only tweak I would suggest is to replace "and high severity" with a comma, as you have done in the next line.